src/components/Router/index.js

```diff
@@ -39,11 +39,23 @@ class CrossDeviceMobileRouter extends Component {
   }
 
   setConfig = (actions) => (data) => {
-    const {token, steps, documentType, step} = data
+    const {token, steps, documentType, step, woopraCookie} = data
+    this.setWoopraCookie(woopraCookie)
     this.setState({token, steps, step})
     actions.setDocumentType(documentType)
   }
 
+  setWoopraCookie = (cookie) => {
```

Comment:
it's much better to do this inside of the Tracker module. It's specific to tracking and not the router. Also, I think you will be able to reference the onfido woopra object without having to access `window`, which should be avoided as much as possible.

Reply:
ok makes sense from a code organisation point of view. Why is accessing `window` bad?

Reply:
because it's a global variable. Anyone can potentially rewrite that property to something else.